core/domain/summary_services_test.py

```diff
@@ -362,3 +363,119 @@ def test_for_featured_explorations(self):
+        self.signup(self.BOB_EMAIL, self.BOB_NAME)
+
+        self.save_new_valid_exploration(self.EXP_ID_1, self.albert_id)
+        self.save_new_valid_exploration(self.EXP_ID_2, self.albert_id)
+        self.save_new_valid_exploration(self.EXP_ID_3, self.albert_id)
+        self.save_new_valid_exploration(self.EXP_ID_4, self.albert_id)
+        self.save_new_valid_exploration(self.EXP_ID_5, self.albert_id)
+        self.save_new_valid_exploration(self.EXP_ID_6, self.albert_id)
+        self.save_new_valid_exploration(self.EXP_ID_7, self.albert_id)
+
+        rights_manager.publish_exploration(self.albert_id, self.EXP_ID_1)
+        rights_manager.publish_exploration(self.albert_id, self.EXP_ID_2)
+        rights_manager.publish_exploration(self.albert_id, self.EXP_ID_3)
+        rights_manager.publish_exploration(self.albert_id, self.EXP_ID_4)
+        rights_manager.publish_exploration(self.albert_id, self.EXP_ID_5)
+        rights_manager.publish_exploration(self.albert_id, self.EXP_ID_6)
+        rights_manager.publish_exploration(self.albert_id, self.EXP_ID_7)
+
+        self.set_admins([self.ADMIN_USERNAME])
+
+    def test_for_top_rated_explorations(self):
+        rating_services.assign_rating_to_exploration(
+            self.bob_id, self.EXP_ID_2, 5)
+        rating_services.assign_rating_to_exploration(
+            self.alice_id, self.EXP_ID_3, 5)
+        rating_services.assign_rating_to_exploration(
+            self.bob_id, self.EXP_ID_3, 4)
+        rating_services.assign_rating_to_exploration(
+            self.bob_id, self.EXP_ID_4, 4)
+        rating_services.assign_rating_to_exploration(
+            self.alice_id, self.EXP_ID_5, 4)
+        rating_services.assign_rating_to_exploration(
+            self.bob_id, self.EXP_ID_5, 3)
+        rating_services.assign_rating_to_exploration(
+            self.bob_id, self.EXP_ID_6, 3)
+        rating_services.assign_rating_to_exploration(
+            self.bob_id, self.EXP_ID_6, 2)
+        rating_services.assign_rating_to_exploration(
+            self.bob_id, self.EXP_ID_7, 1)
+
+        top_rated_exploration_summaries = (
+            summary_services.get_top_rated_exploration_summary_dicts([
+                feconf.DEFAULT_LANGUAGE_CODE]))
+        expected_summary = {
+            'status': u'public',
+            'thumbnail_bg_color': '#a33f40',
+            'community_owned': False,
+            'tags': [],
+            'thumbnail_icon_url': '/images/subjects/Lightbulb.svg',
+            'language_code': feconf.DEFAULT_LANGUAGE_CODE,
+            'id': self.EXP_ID_2,
+            'category': u'A category',
+            'ratings': {u'1': 0, u'3': 0, u'2': 0, u'5': 1, u'4': 0},
+            'title': u'A title',
+            'num_views': 0,
+            'objective': u'An objective'
+        }
+
+        self.assertDictContainsSubset(
+            expected_summary, top_rated_exploration_summaries[0])
+
+        self.assertEqual(7, len(top_rated_exploration_summaries))
```

Comment:
Aren't you going to check the ids of each one, to make sure they're in the correct order?

Also, should have a test with > 8 explorations and make sure only 8 are returned.